dev-requirements.txt

```diff
@@ -1,2 +1,3 @@
 flake8==4.0.1
 black==21.12b0
+websockets==10.1
```

Comment:
Why did you add this here?

Reply:
While trying to build and test robyn, I ran into an issue where the `websockets` dependency hadn't been met. I looked around, and it gets called in `robyn/integration_tests/test_web_sockets.py`. Since it isn't necessary for normal operation, it seemed more appropriate to add it to `dev-requirements.txt`. I suppose it isn't really necessary for this to be included for the feature, but I felt like it was worth slipping it in to help with long term CI/CD.

Reply:
Ah, that makes sense. Thank you! :D